src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -12,7 +12,7 @@ template <typename LogicalDeviceType>
 class IOpenGL_PhysicalDeviceBase : public IPhysicalDevice
```

Comment:
dont use a `_` in the name

Reply:
its meant to mean base for both GL and GLES

Reply:
ah ok, makes sense then